src/make_data/fake-data.R

```diff
@@ -154,36 +154,35 @@ master <-
     #
     # NHS columns
     #
-    nhsnumber = nhs_numbers,
-    dateofbirth = dobs,
-    patientfirstname = first_names,
-    patientothername = other_names,
-    patientsurname = last_names,
-    patientaddress_line1 = str_replace_all(map_chr(addresses, ~ .x$street_address()), "\\n", ""),
-    patientaddress_line2 = map_chr(addresses, ~ .x$city()),
-    patientaddress_line3 = map_chr(addresses, ~ .x$street_name()),
-    patientaddress_line4 = map_chr(places$name_1, ~ ifelse(is.null(.x), NA_character_, .x)),
-    patientaddress_line5 = map_chr(places$county_unitary, ~ ifelse(is.null(.x), NA_character_, .x)),
-    patientaddress_postcode = postcodes,
-    gppracticecode = nhs_gp_practice_codes$code,
-    practice_name = nhs_gp_practice_codes$name,
+    Traced_NHSNUMBER = nhs_numbers,
+    DateOfBirth = dobs,
+    PatientFirstName = first_names,
+    PatientOtherName = other_names,
+    PatientSurname = last_names,
+    PatientAddress_Line1 = str_replace_all(map_chr(addresses, ~ .x$street_address()), "\\n", ""),
+    PatientAddress_Line2 = map_chr(addresses, ~ .x$city()),
+    PatientAddress_Line3 = map_chr(addresses, ~ .x$street_name()),
+    PatientAddress_Line4 = map_chr(places$name_1, ~ ifelse(is.null(.x), NA_character_, .x)),
+    PatientAddress_Line5 = map_chr(places$county_unitary, ~ ifelse(is.null(.x), NA_character_, .x)),
+    PatientAddress_Postcode = postcodes,
+    GPPractice_Code = nhs_gp_practice_codes$code,
+    Practice_NAME = nhs_gp_practice_codes$name,
     contact_telephone = nhs_gp_practice_codes$telephone,
     mobile = ch_phone_number(n, locale = "en_GB"),
     patient_landline = ch_phone_number(n, locale = "en_GB"),
```

Comment:
Should this change to just `landline` as per requirements?

```suggestion
    landline = ch_phone_number(n, locale = "en_GB"),
```

Reply:
Good spot, thanks.